charts/cluster-api-runtime-extensions-nutanix/values.yaml

```diff
@@ -84,6 +84,9 @@ hooks:
 
 helmAddonsConfigMap: default-helm-addons-config
 
+selfHostedRegistry: true
```

Comment:
Could we combine this with `helmRegistryImage` please? So the config looks like:

```
inClusterHelmOCIRegistry:
  enabled: true
  image:
    repository: repository: ghcr.io/nutanix-cloud-native/caren-helm-reg
    tag: ""
    pullPolicy: IfNotPresent
```